### PR TITLE
complete My Account page

### PR DESCRIPTION
**Pull request by chris-saburo**
#123 
- created new page - My Account Page
- added its route
- created user avatar, svg, user account info component

### CODE REVIEW

File: src/components/BaseUserAccountInfo.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<script>
+import BaseInputText from '@/components/BaseInputText.vue';
+
+export default {
+  name: 'BaseUserAccountInfo',
+  components: { BaseInputText },
+  computed: {
+    email: {
+      get() {
+        return this.$store.getters['auth/email'];
```
Comment (atharva3010) [inline src/components/BaseUserAccountInfo.vue:32]: This is ok, but in my opinion it's better to use mapGetters

File: src/views/Settings.vue [hunk 2]
```diff
@@ -60,23 +40,24 @@
...
   name: 'SettingsPage',
-  components: { BaseAppBarHeader, NavigationBottom, LayoutFixedScrollable },
+  components: { BaseAppBarHeader, NavigationBottom, LayoutFixedScrollable, ArrowRightIcon },
   methods: {
     async logout() {
       try {
         await this.$store.dispatch('auth/logout');
         return this.$router.push({ name: 'AuthLogin' });
       } catch (error) {
-        // TODO: snackbar
```
Comment (atharva3010) [inline src/views/Settings.vue:79]: Why have you removed this? TODOs are used by the team to track what all we have plan to do later in code.

**Comment by paulchrisluke** [comment]
since we are not exposing password via the api, we can use a simple button here for reset password.

image

@chris-saburo

File: src/views/Settings.vue (hunk 2 above)
Comment (chris-saburo) [reply src/views/Settings.vue:69]: Done it.

File: src/components/BaseUserAccountInfo.vue (hunk 1 above)
Comment (chris-saburo) [reply src/components/BaseUserAccountInfo.vue:32]: Done it.

**Pull request merged**